samples/geo-ip/lib/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "geoip-component"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+maxminddb = "0.26.0"
+wit-bindgen = "0.41"   
+wit-bindgen-rt = "0.41" 
```

Pretty sure you never need both of these...
```suggestion
wit-bindgen = "0.41"   
```